src/Query/Particular.ThroughputQuery.csproj

```diff
@@ -6,6 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Core" />
```

Comment:
This reference is not required. Any reasons why we add it?

Reply:
Hmm, ok, it seems you are correct. I was unable to find the transient reference previously, but it seems to compile without it. Thanks again!